minari/data_collector/data_collector.py

```diff
@@ -259,6 +260,7 @@ def create_dataset(
                                                                             `ref_max_score` and `expert_policy` can't be passed at the same time. Default to None
             num_episodes_average_score (int): number of episodes to average over the returns to compute `ref_min_score` and `ref_max_score`. Default to 100.
             description (str, optional): description of the dataset being created. Defaults to None.
+            requirements (list of str, optional): list of requirements to load the environment and reproduce the dataset. Defaults to None.
```

Comment:
It may be good to add some very brief documentation about the requirements format used. For example, it may not be obvious that a full pip-style requirement specification is supported.

Reply:
Right, added!